src/net/sf/memoranda/EventsManager.java

```diff
@@ -161,6 +161,25 @@ public static Collection getEventsForMonth(CalendarDate date) {
 		return monthEvents;
 	}
 	
+	/**
+	 * Takes the current selected calendar day and makes a collection of all events
+	 * of that day and following custom day range. US-101.
+	 * 
+	 * @param date 			     current selected date
+	 * @param dayRange 			 number of days to search
+	 * @return customRangeEvent  collection of Events within range
```

Comment:
might should specify it returns a vector as opposed to collection.
shouldn't mention the private method variable customRangeEvent in the javadoc

just say return vector of the events in the range